src/IW/Db/Issue.hs

```diff
@@ -4,48 +4,49 @@
 
 module IW.Db.Issue
        ( getIssues
-       , getIssuesByLabel
+       , getIssuesByLabels
        , upsertIssues
        ) where
 
-import IW.Core.Issue (Issue (..))
+import IW.Core.Issue (Issue (..), Label (..))
+import IW.Core.SqlArray (SqlArray (..))
 import IW.Core.WithId (WithId)
 import IW.Db.Functions (WithDb, executeMany, query, queryRaw)
 
 
--- | Returns all issues in the database
+-- | Returns all issues in the database.
 getIssues :: (WithDb env m) => m [WithId Issue]
 getIssues = queryRaw [sql|
     SELECT id, repo_owner, repo_name, number, title, body, labels
     FROM issues
     LIMIT 100
 |]
 
--- | Returns a list of issues filtered by label name
-getIssuesByLabel :: (WithDb env m) => Text -> m [WithId Issue]
-getIssuesByLabel = query [sql|
+-- | Returns all issues with at least one label in the given list.
+getIssuesByLabels :: (WithDb env m) => [Label] -> m [WithId Issue]
+getIssuesByLabels = query [sql|
     SELECT id, repo_owner, repo_name, number, title, body, labels
-    FROM issues 
-    WHERE ? = ANY (labels)
+    FROM issues
+    WHERE ? && labels
     LIMIT 100
-|] . Only
+|] . Only . SqlArray
```

Comment:
This is a good function to be used with named parameters later. So you can get rid of `Only` and just use:

```haskell
queryNamed [sql|
...
|] [ "labels" =? SqlArray labels ]
```

Reply:
@chshersh I was going to introduce this now, but I think I'm going to open up another pull request and introduce posgresql-simple-named there and just convert some of the functions where it makes sense.

Reply:
@rashadg1030 Yep, it's a good idea to do this in a separate PR 👍